Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/orchestration/AuditTransformer.java

```diff
@@ -35,7 +35,7 @@
         INBOUND, OUTBOUND
```

Comment:
This class really should be moved out to the AuditCore library with the other interfaces.  Should discuss.